src/main/webapp/js/ideas/showIdea.directive.js

```diff
@@ -0,0 +1,29 @@
+(function() {
+	'use strict';
+
+	angular
+		.module('app.directives')
+		.directive('showIdea', showIdea);
+
+	function showIdea() {
+		return {
+			restrict: 'E',
+			replace: true,
+			scope: {
+				idea: '@',
+				photo: '@',				
+				title: '@',
```

Comment:
If idea object is already passed to the directive, may be, other data, like title, description and other, can be obtained from idea object and there is no necessity to pass it to the directive?